mwafrika/teamwork
# #168538826 Add Signin API endpoint

#### What does this PR do?
create API endpoint for Sign in 

#### Description of Task to be completed?
- create test for all the lignin features and condition
- Protect information in the payload from being seen by everyone.
- Use the JWT to encode and decode user information .
- setup the endpoint : 
### /api/v1/auth/signin

#### How should this be manually tested?
After successfully setting up the endpoint, Will use the Postman to test successfully the endpoint end ensure that it should work properly.
 
#### Any background context you want to provide?
- Postman for testing
#### What are the relevant pivotal tracker stories?
#168538826
